e3sm_diags/plot/cartopy/zonal_mean_xy_plot.py

```diff
@@ -1,138 +0,0 @@
-from __future__ import print_function
```

Comment:
This file is note needed anymore. I think this file was accidentally left behind in https://github.com/E3SM-Project/e3sm_diags/pull/752